src/web3/networkConfig.ts

```diff
@@ -232,6 +244,10 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     celoExplorerBaseTokenUrl: CELO_EXPLORER_BASE_TOKEN_URL_ALFAJORES,
     celoExplorerBaseTxUrl: CELO_EXPLORER_BASE_TX_URL_ALFAJORES,
     celoExplorerBaseAddressUrl: CELO_EXPLORER_BASE_ADDRESS_URL_ALFAJORES,
+    blockExplorerBaseTxUrl: {
+      [Network.Celo]: CELO_EXPLORER_BASE_TX_URL_ALFAJORES,
```

Comment:
this is fine...but if you did `[NetworkId['celo-alfajores']]: CELO_EXPLORER_BASE_TX_URL_ALFAJORES` instead then you don't need `networkIdToNetwork`? i'm not sure, all these mappings make me confused

Reply:
also, maybe we can change `CELO_EXPLORER_BASE_URL_ALFAJORES` and `CELO_EXPLORER_BASE_URL_MAINNET` to celo scan? :D

Reply:
thanks for the feedback about confusion! it makes sense.
I refactored it a bit to make more straightforward.

as for moving to CeloScan, would you mind if I bring that in a separate PR immediately after this one is merged?
it has to deal with other references and translations too, and I would like to have those changes clear, not messed up with all changed in here.